app.css

```diff
@@ -1,7 +1,51 @@
 /* apply a natural box layout model to all elements, but allowing components to change */
 html {
   box-sizing: border-box;
+  backgound: white;
+  color: rgb(218, 218, 218);
+  font-size: 1rem;
+  font-family: sans-serif;
+
 }
 *, *:before, *:after {
   box-sizing: inherit;
 }
+
+body {
+  max-width: 1000px;
+}
+
+strong {
+  color: #000000;
+}
+
+hr {
+  margin: 0;
+}
+
+.picture {
+  width: 300px;
+  border: solid rgb(218, 218, 218) 1px;
+  padding: 10px;
+}
+
+.section {
+  float: left;
+  padding: 10px;
+}
+
+.number {
+  display: inline-block;
+}
+
+.author-avatar {
+  display: inline;
```

Comment:
Since this is floated, you probably don't need `display: inline` but this may change some height things so you will want to check on that.